_templates/component/new/wdio-spec.js.t

```diff
@@ -0,0 +1,13 @@
+---
+to: src/components/<%= name %>/test/wdio/default.js
```

Comment:
Let's talk more about how we want to structure each component directory. I usually lean away from creating a bunch of deep nesting, but I understand there may be a good argument for it.

Reply:
Would prefer that this generates a typescript file if possible.

Reply:
the difficulty with typescript is the visual regression service doesn't have types associated with it, and webdriver didn't add types until it's webdriver 5.0 release. For typescript, it'd be on us to manage those type definitions.

Reply:
Yeah that makes sense. In these cases I don't mind adding a `.d.ts` file to supplement the lack of type defs to maintain the other benefits of it. It has ramifications from a linting/formatting perspective on top of the ability to write preventable bugs. Happy to talk through some of these challenges in person if that's helpful. This is really good work, just trying to make sure it fits with the current thrust. 👍

Reply:
I looked around online, and found a list of type definitions to import for webdriverio. In addition, I was able to get the configuration file as well converted over to Typescript. There are a lot of any declarations within the types I found for webdriver, but I tried to only stick to changing the ones that we currently use in the app. You can see the commits here.

https://github.com/cremalab/crema-app-web/pull/17/commits/ebc80fe4a70626cadd87b31b330ba363bd4d3ad6
https://github.com/cremalab/crema-app-web/pull/17/commits/67d70b93092d7817f7031166a360006b970d3c86
https://github.com/cremalab/crema-app-web/pull/17/commits/e8756360fc10ad9ab7024f6f6576562177c4942d
https://github.com/cremalab/crema-app-web/pull/17/commits/54363636042eafd3f4d7c3dbc699b0a607724f09
https://github.com/cremalab/crema-app-web/pull/17/commits/31b295c6697f22a96302c9a8b11e69c9cbc571e4
https://github.com/cremalab/crema-app-web/pull/17/commits/0ed75f18093d50236f8fc586290d4f9622bd0c63